magnet_coils.py

```diff
@@ -0,0 +1,350 @@
+import cubit
+import numpy as np
+import logging
+
+
+# Get logger defined in parametric_stellarator.py
+logger = logging.getLogger('log')
+
+
+def orient_rectangle(path_origin, surf_id, t_vec, norm, rot_axis, rot_ang_norm):
+    """Orients rectangular cross-section in the normal plane such that its
+    thickness direction faces the origin.
+    
+    Arguments:
+        path_origin (int): index of initial point in filament path.
+        surf_id (int): index of cross-section surface.
+        t_vec (list of float): cross-section thickness vector.
+        norm (list of float): cross-section normal vector.
+        rot_axis (list of float): axis about which to rotate the cross-section.
+        rot_ang_norm (float): angle by which cross-section was rotated to align
+            its normal with the initial point tangent (deg).
+    """
+    # Determine orientation of thickness vector after cross-section was
+    # oriented along filament origin tangent
+
+    # Compute part of thickness vector parallel to rotation axis
+    t_vec_par = np.inner(t_vec, rot_axis)*rot_axis
+    t_vec_par = t_vec_par/np.linalg.norm(t_vec_par)
+    # Compute part of thickness vector orthogonal to rotation axis
+    t_vec_perp = t_vec - t_vec_par
+    t_vec_perp = t_vec_perp/np.linalg.norm(t_vec_perp)
+
+    # Compute vector othogonal to both rotation axis and orthogonal
+    # part of thickness vector
+    orth = np.cross(rot_axis, t_vec_perp)
+    orth = orth/np.linalg.norm(orth)
+
+    # Convert angle from degress to radians
+    rot_ang_norm = np.deg2rad(rot_ang_norm)
+    
+    # Determine part of rotated vector parallel to original
+    rot_par = np.cos(rot_ang_norm)
+    # Determine part of rotated vector orthogonal to original
+    rot_perp = np.sin(rot_ang_norm)
+
+    # Compute orthogonal part of thickness vector after rotation
+    t_vec_perp_rot = rot_par*t_vec_perp + rot_perp*orth
+    # Compute thickness vector after rotation
+    t_vec_rot = t_vec_perp_rot + t_vec_par
+    t_vec_rot = t_vec_rot/np.linalg.norm(t_vec_rot)
+
+    # Orient cross-section in its plane such that it faces the global origin
+
+    # Extract initial path point
+    pos = cubit.vertex(path_origin).coordinates()
+
+    # Project position vector onto cross-section
+    pos_proj = pos - (np.inner(pos, norm)/np.inner(pos, pos))*norm
+    pos_proj = pos_proj/np.linalg.norm(pos_proj)
+
+    # Compute angle by which to rotate cross-section such that it
+    # faces the origin
+    dot_prod = np.inner(pos_proj, t_vec_rot)
+    rot_ang_orig = np.arccos(dot_prod)
+
+    # Convert angle from radians to degrees
+    rot_ang_orig = np.rad2deg(rot_ang_orig)
+    
+    # Re-orient rotated cross-section such that thickness vector
+    # faces origin
+    cubit.cmd(
+        f'rotate Surface {surf_id} angle {rot_ang_orig} about  origin 0 0 0 ' +
```

Comment:
Looking to streamline... 

Can we put the `rad2deg` conversion inline here?
```suggestion
        f'rotate Surface {surf_id} angle {np.rad2deg(rot_ang_orig)} about  origin 0 0 0 ' +
```